src/IncidentReport.js

```diff
@@ -171,10 +171,9 @@ class IncidentReport extends Component {
                 .then((text) => {
                     alert(text.error);
                 });
-            } else {
+            } else if (response.status === 201) {
               form.reset();
               update(true);
```

Comment:
What does 201 signify? Also, is update(true) what you meant? Not this.updateIncidentStatus(true)?